File: api/src/EventHub.Domain/ReadModels/OrganizerDto.cs
namespace EventHub.Domain.ReadModels;

/// <summary>Organisateur exposé au back office (GET /api/admin/organizers).</summary>
public sealed record OrganizerDto
{
    public required Guid Id { get; init; }
    public required string Name { get; init; }
    public string? ContactEmail { get; init; }
}

File: backoffice/src/app/features/organizers/models/organizerDto.ts
/** Organisateur (GET /api/admin/organizers). */
export interface OrganizerDto {
  id: string;
  name: string;
  contactEmail?: string | null;
}

/** Payload create/update (POST/PATCH /api/admin/organizers). */
export interface OrganizerRequest {
  name: string;
  contactEmail?: string | null;
}
